dist/build.mjs

```diff
@@ -194,8 +262,13 @@ class Build {
   }
 }
 
+let configParams = parseClArgs()
+const configModule = 'file:///' + path.posix.join(process.cwd().replace(/\\/g, '/'), 'build/', configParams.configFileName)
+
 import(configModule)
   .then(config => {
-    let build = new Build(config)
+    let build = new Build(configParams, config)
     build.runModules()
   }).catch(e => console.error(`Cannot resolve a config file module ${configModule}: ${e}`))
+
+export { generateBuildNumber }
```

Comment:
why do we have to export generateBuildNumber here?

Reply:
Oh, that's the remains of the old code where it was imported directly from build.mjs. But as it was causing other imports of build.mjs to be loaded I moved it to a separate file. It is not needed any more and I removed it.